internal/app/handlers/required_interfaces.go

```diff
@@ -0,0 +1,25 @@
+package handlers
+
+import (
+	"context"
+	"yandex_gophermart/pkg/entities"
+)
+
+//go:generate mockgen -destination=mocks/mock_interfaces.go yandex_gophermart/internal/app/handlers StorageInt,JWTHelperInt
+
+type StorageInt interface {
+	SaveUser(login string, passwordHash string, passwordSalt string, ctx context.Context) (int, error) //int - ID
```

Comment:
```suggestion
	SaveUser(ctx context.Context, login string, passwordHash string, passwordSalt string) (int, error) //int - ID
```
Go-style - всегда передавать контекст первым параметром. Касается всех функций из интерфейса.

Reply:
Готово